Source/WebKit/UIProcess/Inspector/mac/WKInspectorViewController.mm

```diff
@@ -95,6 +95,7 @@ - (WKWebView *)webView
     if (!_webView) {
         NSRect initialFrame = NSMakeRect(0, 0, WebKit::WebInspectorUIProxy::initialWindowWidth, WebKit::WebInspectorUIProxy::initialWindowHeight);
         _webView = adoptNS([[WKInspectorWKWebView alloc] initWithFrame:initialFrame configuration:self.webViewConfiguration]);
+        [_webView setInspectable:YES];
```

Comment:
should we do this based on whether or not the "can inspect the inspector" setting is set?

Reply:
I think it is useful on macOS for a Web Inspector to be inspectable out-of-process even if the setting hasn't been toggled, since it might be in such a bad state that toggling the setting isn't practical.

Reply:
hmm i do worry that this might confuse developers tho if they happen to have both Safari and STP installed, as one might not realize that Web Inspector is a webpage

Reply:
Safari and STP today aren't able to inspect the other, and I expect that to remain true.